extensions/Khronos/KHR_materials_cmnConstant/README.md

```diff
@@ -0,0 +1,111 @@
+
+## Overview
+
+This extension defines an unlit ("constant") shading model for use in glTF 2.0 materials, as an alternative to the Physically Based Rendering (PBR) shading models provided by the core specification. In mobile devices with limited resources, unlit materials offer a performant alternative to higher-quality shading models.
+
+## Extending Materials
+
+The common Constant material is defined by adding the `KHR_materials_cmnConstant` extension to any glTF material. When present, the extension indicates that a material should use the Constant shading model. Unrelated default values in the default PBR shading model, such as `diffuseFactor`, should be ignored.
+
+```
+{
+    "materials": [
+        {
+            "name": "MyUnlitMaterial",
+            "emissiveFactor": [ 0.2, 0.1, 0.0 ],
+            "extensions": {
+                "KHR_materials_cmnConstant": {}
+            }
+        }
+    ]
+}
+```
+
+No properties are defined within the extension object — it is simply a boolean flag indicating use of an unlit shading model.
+
+### Terms
+
+
+
+### Definition
+
+The Constant material model describes a constantly shaded surface that is independent of lighting. The material is defined only by properties already
+present in the [glTF 2.0 material specification](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#material): no new properties are added by this extension.
+
+Additional properties are allowed, but may lead to undefined behaviour in conforming viewers.
+
+The material is intended for unlit shading. Color is calculated as:
+
+```
+color = <emissiveTerm> + <occlusionTerm> * <aL>
+```
+
+Each term is defined as follows:
+
+* `emissiveTerm` – emissiveFactor * emissiveTexture
+    *  [*From core glTF material specification.*](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#additional-maps)
+* `occlusionTerm` – occlusionFactor * occlusionTexture
+    *  [*From core glTF material specification.*](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#additional-maps)
+* `aL` – Ambient light
+    * *Constant sum of all ambient light contributions from the scene.*
+
+> **Implementation Note**: The constant shading model is visually identical to
+> Lambert or Blinn-Phong models in which all unused factors are set to zero.
+> Accordingly, engines that already implement these models may use them to
+> emulate simpler shading models if desired.
+
+### Example
+
+The following example defines a Constant material with an emissive red color.
+
+```json
+"materials": [
+    {
+        "name": "red_constant_material",
+        "emissiveFactor": [ 1.0, 0.0, 0.0 ],
+        "extensions": {
+            "KHR_materials_cmnConstant": {}
+        }
+    }
+]
+```
+
+### Extension compatibility and fallback materials
+
+When possible, authoring tools should provide a fallback material definition
+and mark the `KHR_materials_cmnBlinnPhong` extension as optional. By doing so,
```

Comment:
This should reference KHR_materials_cmnConstant and not BlinnPhong.

Reply:
✅